### DISCUSSION TITLE
CPU percentages tab to go all way down

### DISCUSSION THREAD

**Discussion by joe2xyz** [ClementTsang/bottom#1403] [Q&A]
Hi folks,

**Thank you** @ClementTsang  and all contributors for this **amazing project**. So much better than any other htop-like programs.

I'm playing with the configuration and I managed to turn some default tabs on and off but I can't figure out how to do something like this:

![screenshot](https://github.com/ClementTsang/bottom/assets/122208221/0124dfc8-d2b3-43fc-83e1-613c891def39)

(I want CPU percentages tab to go all way down)

Can somebody please show me some an example configuration with such panel?

### Thank you all!

Joe

**Comment by ClementTsang** [comment]
Hmm... I don't think that's currently doable but in theory I could make it doable if I decouple the legend as a separate widget.

**Comment by joe2xyz** [comment]
Thank you for the answer.

The reason I asked is that I have 16 cores and I'd like to see them all in a small window. Basically I want to see how many are at 100% and how many are doing nothing. I thought that the above configuration would be the easiest way to achieve that but there are probably other ways how to do that - for example multiple columns:

![columns](https://github.com/ClementTsang/bottom/assets/122208221/aa66ac8b-2d5b-44de-bbc8-ecff6bf531c6)

Thank you.

**Comment by mert-kurttutan** [comment]
I would also very muck like that since it is very useful to monitor multi threaded applications.
At the moment, I dont have to help, I can let you know when I have it.
